packages/webpack/src/config/client.js

```diff
@@ -86,6 +82,12 @@ export default class WebpackClientConfig extends WebpackBaseConfig {
           .map(name => name.replace(/[/\\]/g, '.').replace(/_/g, '').replace('pages.', ''))
           .join('~')
 
+        // Fixes https://github.com/nuxt/nuxt.js/issues/7665
+        // TODO: We need a reproduction for this case (test/fixtures/shared-chunk)
```

Comment:
@simllll @aldarund I've added a TODO item as couldn't reproduce the issue. Would be awesome if we can find a reproduction for fixtures. And sorry for issues that happened due to this name change...

Reply:
Nuxt 2.13.13 was broken trying to load chunks like `.xxx.js`. This has fixed it. Thanks @simllll and @pi0